### PR TITLE
add security groups for app server and database

### PR DESCRIPTION
**Pull request by BrianGreenhill**
addresses #7

### CODE REVIEW

File: infra/tf/vpc/sg.tf [hunk 1]
```diff
@@ -0,0 +1,36 @@
+resource "aws_security_group" "app_servers" {
+  name        = "app_servers"
+  description = "Allow inbound web traffic"
+  vpc_id      = "${module.vpc.vpc_id}"
+
+  ingress {
+    description = "Port 80 from world to application servers"
+    from_port   = 80
```
Comment (prasadpilla) [inline infra/tf/vpc/sg.tf]: @BrianGreenhill Can you add port 22 for SSH?

Comment (prasadpilla) [reply infra/tf/vpc/sg.tf]: Or @BrianGreenhill what's the best practice here?

Comment (BrianGreenhill) [reply infra/tf/vpc/sg.tf]: we can allow it from any ip as a starting point. a best practice that i've seen used in the past is to create a bastion host that can be used as a jump server to access business critical resources.

Comment (prasadpilla) [reply infra/tf/vpc/sg.tf]: @BrianGreenhill Okay sure, sounds good.

**Pull request merged**